setup.py

```diff
@@ -0,0 +1,26 @@
+"""
+This is a setup.py script generated by py2applet
+
+Usage:
+    python setup.py py2app
+"""
+
+from setuptools import setup
+
+APP = ["minesweeper.py"]
+DATA_FILES = [
+    "doomguy.gif",
+    "bomb_small.png",
+    "bomb.png",
+    "button_small.png",
+    "flag_small.png",
+    "pressed_button_small.png",
+]
```

Comment:
```suggestion
DATA_FILES = glob.glob("*.gif") + glob.glob("*.png")
```

Needs `import glob`.